pkg/operator/controller/controller.go

```diff
@@ -95,10 +101,24 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 		ingress = nil
 	}
 
-	caSecret, err := r.ensureRouterCACertificateSecret()
-	if err != nil {
-		errs = append(errs, fmt.Errorf("failed to ensure CA secret: %v", err))
-	} else {
+	caSecretName := cacert.CASecretName(r.Namespace)
```

Comment:
As outlined in the PR description, everything related to the certs can eventually be stripped out of this controller; trying to get there one small step at a time